mgmtsystem/__manifest__.py

```diff
@@ -39,5 +23,5 @@
         'views/res_config.xml'
     ],
     "demo": [],
-    'installable': False,
+    'installable': True,
 }
```

Comment:
@max3903 
IMO mgmtsystem_manual and document_page_work_instruction depend both on document_page_approval and document_page_procedure depends on document_page_work_instruction.
document_page_approval depends on document_page